orion/sites/admin/src/achieve/achieveEdit.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+require('./achieveEdit.styl');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
+
+export class AchieveEditPage extends React.Component {
+constructor(props) {
+  super(props);
+  this.state = {
+    isEdit: false,
+    showDeleteModal: false,
+    showSaveModal: false,
+    inputYear: "",
+    inputMessage: "",
+  };
+}
+
+  render() {
+    const title = "Edit Achievement";
+    let deleteButton = <div></div>;
+    return (
+      <div id="view-achieve-edit">
+        <h2>{title}</h2>
+        <h4>Year</h4>
+        <input value={this.state.inputYear}
+                onChange={(e) => this.handleChange(e, "inputYear")}/>
+        <h4>Message</h4>
+        <input value={this.state.inputMessage}
+                onChange={(e) => this.handleChange(e, "inputMessage")}/>
+        <div className="buttons">
+          <button className="btn-save" onClick={this.onClickSave}>Save</button>
```

Comment:
Clicking on this button should invoke a function called `onClickSave`.

Can you define a function in this React Component called `onClickSave`? You can just have a very simple `console.log("save button clicked")` message inside it.
Do the same for `onClickCancel`.